### PR TITLE
[WIP][Documentation] Customizing credit memo model itself

### PR DESCRIPTION
**Pull request by pamil**
Basic customization of RefundPlugin templates.

### CODE REVIEW

**Review by GSadee — APPROVED** [review]
I am wondering about the last example, shouldn't we describe the overriding credit memo by adding some extra field to this entity instead of rendering extra data from the different controller? 🤔

**Review by AdamKasp — APPROVED** [review]

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    .. code-block:: html
+
+        <style>
+            /* ... */
+            .credit-memo table tr.heading td { background: #0d71bb; border-bottom: 1px solid #ddd; font-weight: bold; }
+            /* ... */
+        </style>
+
+Additional customer data
```
Comment (CoderMaggie) [inline docs/cookbook/payments/customizing-credit-memo.rst:24]:
```suggestion
Displaying additional Customer's data on Credit Memo
```

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+        <td>
+            {{ 'sylius_refund.ui.buyer'|trans([], 'messages', creditMemo.localeCode) }}<br/>
+            <strong>{{ from.fullName }} </strong><br/>
+            <!-- ... -->
+            {{ creditMemo.order.customer.phoneNumber }}<br/>
+            <!-- ... -->
+        </td>
+
+Additional line item data (such as gross unit price)
```
Comment (CoderMaggie) [inline docs/cookbook/payments/customizing-credit-memo.rst:44]:
```suggestion
Displaying additional line item data (such as gross unit price) on Credit Memo
```

File: docs/cookbook/payments/customizing-credit-memo.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+**3.** Add missing translations for newly added string in ``translations/messages.en.yml``:
+
+    .. code-block:: yaml
+
+        app:
+            ui:
+                unit_gross_price: Unit gross price
+
+Additional unconnected data
```
Comment (CoderMaggie) [inline docs/cookbook/payments/customizing-credit-memo.rst:82]:
```suggestion
Displaying additional elements on Credit Memo
```

**Review by CoderMaggie — COMMENTED** [review]
Cool, however, I do agree with @GSadee's comment.

**Comment by pamil** [comment]
There's an issue with overriding an entity - it does not work currently: https://github.com/Sylius/RefundPlugin/issues/278

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "customizing-credit-memo"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Pull request closed without merging**